jvb/src/main/kotlin/org/jitsi/videobridge/rest/binders/ServiceBinder.kt

```diff
@@ -42,9 +36,6 @@ class ServiceBinder(
             bind(statsCollector).to(StatsCollector::class.java)
         }
         bind(xmppConnection).to(XmppConnection::class.java)
-        // These are still suppliers rather than direct instances because that's what
-        // Jicoco requires
-        bind(JvbHealthCheckServiceSupplier(healthChecker)).to(HealthCheckServiceSupplier::class.java)
-        bind(JvbVersionServiceSupplier(versionService)).to(VersionServiceSupplier::class.java)
```

Comment:
So we no longer need version service here because Jicoco is handling that automatically?

Reply:
It is not passed as a constructor argument to the class that handles `/about/version` instead of using `@Inject`. And that's the only jetty class that uses the version.

Reply:
Ah! Got it.  I was thinking this was registering the endpoint, forgot it was just binding the services.